.github/workflows/issues.yml

```diff
@@ -55,7 +53,7 @@
           echo "The following JIRA ticket has been created: ${JIRA_TICKET_ID}"
           echo "jira-ticket-id=${JIRA_TICKET_ID}" >> "${GITHUB_OUTPUT}"
       - name: Add comment
-        uses: peter-evans/create-or-update-comment@71345be0265236311c031f5c7866368bd1eff043
+        uses: peter-evans/create-or-update-comment@v4
```

Comment:
In SDK we intend to pin github changes to avoid vendor attacks.
In this case this action deals with critical ceredentials so we should pin it.